src/components/Map.js

```diff
@@ -1,11 +1,8 @@
 import React, { Component } from "react";
 import { GoogleMap, LoadScript, Marker } from "@react-google-maps/api";
-import logo from "../img/logo.svg";
 
 class Map extends Component {
-  constructor(props) {
-    super(props);
```

Comment:
Same thing here, map is a child component of steakhouseList, it requires props to receive the locations to pin them